tests/QssTestDefs.js

```diff
@@ -111,6 +111,75 @@ gpii.tests.qss.testPspAndQssVisibility = function (app, params) {
     );
 };
 
+gpii.tests.qss.getFocusedElementIndex = function () {
+    // Note that the elements will be returned in the order in which they appear in the DOM.
+    var qssButtons = jQuery(".fl-qss-button"),
+        focusedElement = jQuery(".fl-focused")[0];
+    return jQuery.inArray(focusedElement, qssButtons);
+};
+
+gpii.tests.qss.pressKey = function (key, modifiers) {
+    return {
+        funcName: "gpii.tests.qss.simulateShortcut",
+        args: [
+            "{that}.app.qssWrapper.qss.dialog",
+            {
+                key: key,
+                modifiers: modifiers
+            }
+        ]
+    };
+};
+
+gpii.tests.qss.assertFocusedElementIndex = function (expectedIndex) {
+    return {
+        task: "gpii.test.executeJavaScriptDelayed",
+        args: [
+            "{that}.app.qssWrapper.qss.dialog",
+            gpii.test.toIIFEString(gpii.tests.qss.getFocusedElementIndex),
+            100
+        ],
+        resolve: "jqUnit.assertEquals",
+        resolveArgs: [
+            "The correct button in the QSS is focused",
+            expectedIndex,
+            "{arguments}.0"
+        ]
+    };
+};
+
+var qssSettingsCount = 11;
```

Comment:
This looks a bit fragile to me but it is probably ok for the tests as the their settings list is controlled.

Reply:
Yes, I also thought about this. However, getting the actual number of settings within the test sequence will involve more code which will pollute the test cases themselves. That is why I preferred to have this variable.